framework/view/view.gotext

```diff
@@ -0,0 +1,34 @@
+package view
+
+// GENERATED. DO NOT EDIT.
+
+
+{{- if $.Imports }}
+
+import (
+	{{- range $import := $.Imports }}
+	{{$import.Name}} "{{$import.Path}}"
+	{{- end }}
+)
+{{- end }}
+
+{{- if not $.Flag.Embed }}
+// Load the view server. Files are linked rather than embedded.
+func Load(client budclient.Client) Server {
+	return viewrt.Proxy(client)
```

Comment:
Requests for client-side built views are now proxied to the bud server to be built and rendered